janusgraph-core/src/main/java/org/janusgraph/core/channelizers/JanusGraphChannelizer.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 JanusGraph Authors
```

Comment:
@mbrukman What is the final word here?

Reply:
If this is a brand new file created in 2018 (which is what it appears to be since it's not branched and appears to be added in this PR), it should be (c) 2018.

If this is a modification of an existing file which as created in 2017 and (c) 2017, there's no need to update the year of copyright, just keep it as it was originally.